src/website/views/javascript/popover.pug

```diff
@@ -0,0 +1,296 @@
+---
+title: Popover
+---
+
+p.-text
+  | Popovers provide an contextual space, attached to an element and in an upper z-index layer, where you can
+  | include any kind of HTML code related to the trigger.
+
+h3 Base
+p.-text
+  | To render a popover, add your HTML code to a <code>data-popover-content</code> attribute. Then,
+  | you can initialize it with the chi.popover factory method.
+.example.-mb--3
+  .-p--3
+    button(data-popover-content='<div class="-px--2"><h4>Popover!</h4><p>This is a popover content triggered by a button. </p></div>').a-btn.-mr--2 Popover
```

Comment:
@dani-cl-madrid this doesn't appear to be working

Reply:
Restored initialization of that example.